core/network/src/protocol/sync.rs

```diff
@@ -118,26 +131,49 @@ pub(crate) enum AncestorSearchState<B: BlockT> {
 }
 
 #[derive(Copy, Clone, Eq, PartialEq, Debug)]
```

Comment:
This is nit-picking, but shouldn't we be consistent in regard to "derive-on-top-of-doc" versus "derive-underneath-doc"?

Reply:
It would indeed be nice to be consistent on that. Maybe we add a rule to the style guide?